ruby-training/trainees/shareni/test_katas/04/converter/converter.rb

```diff
@@ -0,0 +1,10 @@
+class Converter
+    def convert_number_to_reverse_array(original_number)
```

Comment:
The "convert" action is already alluded to in the class name... I'd suggest a method name such as `to_reverse_array(original_number)`. This actually matches Ruby's own casting method names (.to_s, .to_a, etc.) in the `to_` part. `Converter.to_reverse_array(number)` reads a little better than the redundant `converter.convert...`